<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="26571" started-at="2018-08-16T05:55:48Z" finished-at="2018-08-16T05:56:15Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="26571" started-at="2018-08-16T05:55:48Z" finished-at="2018-08-16T05:56:15Z">
      <class name="com.ecomtest.qa.base.BaseTest">
        <test-method status="PASS" signature="loadProperties()[pri:0, instance:com.ecomtest.qa.base.BaseTest@17cd7a0]" name="loadProperties" is-config="true" duration-ms="11" started-at="2018-08-16T11:25:48Z" finished-at="2018-08-16T11:25:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loadProperties -->
      </class> <!-- com.ecomtest.qa.base.BaseTest -->
      <class name="com.ecomtest.qa.scripts.LoginTest">
        <test-method status="PASS" signature="initialization()[pri:0, instance:com.ecomtest.qa.scripts.LoginTest@4f30ec]" name="initialization" is-config="true" duration-ms="10368" started-at="2018-08-16T11:25:49Z" finished-at="2018-08-16T11:25:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialization -->
        <test-method status="PASS" signature="loginTest()[pri:0, instance:com.ecomtest.qa.scripts.LoginTest@4f30ec]" name="loginTest" duration-ms="5185" started-at="2018-08-16T11:25:59Z" finished-at="2018-08-16T11:26:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.ecomtest.qa.scripts.LoginTest@4f30ec]" name="tearDown" is-config="true" duration-ms="1111" started-at="2018-08-16T11:26:04Z" finished-at="2018-08-16T11:26:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="initialization()[pri:0, instance:com.ecomtest.qa.scripts.LoginTest@4f30ec]" name="initialization" is-config="true" duration-ms="8225" started-at="2018-08-16T11:26:05Z" finished-at="2018-08-16T11:26:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialization -->
        <test-method status="FAIL" signature="loginTest1()[pri:0, instance:com.ecomtest.qa.scripts.LoginTest@4f30ec]" name="loginTest1" duration-ms="2" started-at="2018-08-16T11:26:14Z" finished-at="2018-08-16T11:26:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.ecomtest.qa.scripts.LoginTest.loginTest1(LoginTest.java:36)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest1 -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.ecomtest.qa.scripts.LoginTest@4f30ec]" name="tearDown" is-config="true" duration-ms="921" started-at="2018-08-16T11:26:14Z" finished-at="2018-08-16T11:26:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.ecomtest.qa.scripts.LoginTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
